ADOPTERS.md

```diff
@@ -217,3 +217,4 @@ _You can do this by using the [Adopter form](https://info.backstage.spotify.com/
 | [Alaska Airlines](https://alaskaair.com)                                    | [@swerdick](https://github.com/swerdick)                                                                                                                                                                                                                         | Backstage is the developer portal for our 'software delivery platform'.  Consolidating developer tools to one place, and providing automation to make it easy for developers to create and deploy applications to Kubernetes
 | [Loft](https://loft.com.br)                                    | [Squad DevTools](mailto:squad_devtools@loft.com.br)                                                                                                                                                                                                                         | We're using Backstage to give visibility and promote ownership of all our applications, resources and tools. Now moving to use it as a Developer Portal to create applications, AWS resources etc.                                                                                                                                                                                                                                                                                                                                                                                           |
+| [Trifork](https://trifork.com)                                    | [Squad DevTools](mailto:squad_devtools@loft.com.br)                                                                                                                                                                                                                         | We're using Backstage to give visibility and promote ownership of all our applications, resources and tools. Now moving to use it as a Developer Portal to create applications, AWS resources etc.                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Looks like a bit of copy paste slipped through here 😅

Reply:
Thanks! VSCode has not saved the file before commiting